build/AllLinuxDistrosNativeInstaller.targets

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)/LinuxDistrosNativeInstaller.props" />
+
+  <Target Name="AllLinuxDistrosNativeInstaller" DependsOnTargets="Prepare;Compile;RunAllSandBoxAndPackage;MultiPublish">
```

Comment:
Should this be BuildAndPublishAllLinuxDistrosNativeInstallers?

Reply:
Fixed